fcl-config.cmake.in

```diff
@@ -0,0 +1,20 @@
+# Generated by CMake @CMAKE_VERSION@ for @PROJECT_NAME@.
+
+@PACKAGE_INIT@
+
+set(FCL_VERSION "@FCL_VERSION@")
+set(FCL_ABI_VERSION "@FCL_ABI_VERSION@")
+
+set_and_check(FCL_INCLUDE_DIRS "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")
+set(FCL_LIBRARIES @PROJECT_NAME@)
+set_and_check(FCL_LIBRARY_DIRS "@PACKAGE_CMAKE_INSTALL_LIBDIR@")
```

Comment:
There is any reason to define the `FCL_INCLUDE_DIRS` and `FCL_LIBRARY_DIRS` variables? If necessary, these can be  easily be obtained via the `fcl` imported target.

Reply:
Partly tradition, but also makes for a clearer error message when the path is accidentally cleaned IMO.